docs/architecture/adr-047-handling-evidence-from-light-client.md

```diff
@@ -26,6 +27,11 @@ type ConflictingHeadersEvidence struct {
 }
 ```
 
+_Remark_: Theoretically, only the header, which differs from what a full node
+has, needs to be sent. But sending two headers a) makes an evidence easily
+verifiable b) simplifies retransmission (since the full node does not have to
+query each receiver about which header it has: A or B)
```

Comment:
let me know if you (the reader) disagree

Reply:
Is `ConflictingHeadersEvidence` sent between other full nodes though? My understanding was that this composite evidence was only sent from light clients to full nodes and then the evidence was immediately broken down and then sent to the other nodes. Regardless, two headers will need to be sent because the light client will send `ConflictingHeadersEvidence` to witnesses that it doesn't know which header the witness has.

Reply:
> Regardless, two headers will need to be sent because the light client will send ConflictingHeadersEvidence to witnesses that it doesn't know which header the witness has.

We could ask a witness, but not sure it's worth it. I'll remove the (b) sentence. Thanks!